simple_pid/PID.py

```diff
@@ -13,6 +14,17 @@ def _clamp(value, limits):
     return value
 
 
+def pi_clip(angle):
+    """Transform the angle value to a [-pi, pi) range."""
+    if angle > 0:
+        if angle > math.pi:
+            return angle - 2 * math.pi
+    else:
+        if angle < -math.pi:
+            return angle + 2 * math.pi
+    return angle
+
+
```

Comment:
I don't think it's the right way to go to include the mapping function in the library. The library is on purpose small and simple, and providing pre-defined mapping functions is not within the scope of this library. I think it's enough to include it as an example in the readme, but what would be even better would be to make an actual complete example under examples/ in the project root directory that a user could run and which clearly shows how the error mapping feature works. I wont make that a strict requirement for merging this PR but if you could make such an example I would really appreciate it!

Reply:
Yes, you are right, I added the function only in the test file.